services/webnn/ort/graph_builder_ort.cc

```diff
@@ -240,15 +243,26 @@ std::string GraphBuilderOrt::GetOperandName(uint64_t operand_id) {
   }
 }
 
-std::string GraphBuilderOrt::CreateInitializerAsRawData(
-    base::span<const uint32_t> shape,
-    base::span<const uint8_t> data,
-    OperandDataType data_type) {
+template <typename T>
+std::string GraphBuilderOrt::CreateInitializer(base::span<const uint32_t> shape,
+                                               base::span<const T> data,
+                                               OperandDataType data_type,
+                                               bool used_for_shape_inference) {
   std::string name = GetInsertedOperandName(next_operand_id_);
   OperandInfo operand_info{name, data_type, shape};
+  size_t byte_size = sizeof(T) * data.size();
```

Comment:
Could it be
```suggestion
  size_t byte_size = data.as_bytes().size();
```

Reply:
I guess you mean `base::as_bytes(data).size()`. I can't use it, there is compilation error...

Reply:
Ack.